simplefs/simplefs.go

```diff
@@ -529,6 +529,7 @@ func (k *SimpleFS) listRecursiveToDepth(opID keybase1.OpID,
 			if err != nil {
 				return err
 			}
+			d.Name = finalElem
```

Comment:
This isn't technically needed, right?  `setStat` should already set it, I think.  (Am fine with having it here too for clarity, though.)

Reply:
Correct, technically not needed. Originally I thought it'd give absolute-ish paths (`/private/jdoe/a`), but it doesn't, making this line extraneous.